fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BottomToolbarContainerView.kt

```diff
@@ -33,11 +41,22 @@ class BottomToolbarContainerView(
     navigationItems: List<ActionItem> = NavigationItems.defaultItems,
     androidToolbarView: View? = null,
     menuButton: MenuButton,
+    browserStore: BrowserStore,
+    browsingModeManager: BrowsingModeManager,
 ) {
 
     init {
         val composeView = ComposeView(context).apply {
             setContent {
+                val isPrivate = browsingModeManager.mode.isPrivate
+                val tabCount = browserStore.observeAsState(0) { browserState ->
```

Comment:
You can also get the browserStore via `components.core.store` since `components` is exposed in @composable.